sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/QuestionAnsweringClient.cs

```diff
@@ -52,6 +52,30 @@ public QuestionAnsweringClient(Uri endpoint, AzureKeyCredential credential, Ques
             _restClient = new(Diagnostics, Pipeline, Endpoint, Options.Version);
         }
 
+        /// <summary> Initializes a new instance of QuestionAnsweringClient. </summary>
+        /// <param name="endpoint"> Supported Cognitive Services endpoint (e.g., https://&lt;resource-name&gt;.cognitiveservices.azure.com). </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public QuestionAnsweringClient(Uri endpoint, TokenCredential credential) : this(endpoint, credential, new QuestionAnsweringClientOptions())
+        {
+        }
+
+        /// <summary> Initializes a new instance of QuestionAnsweringClient. </summary>
+        /// <param name="endpoint"> Supported Cognitive Services endpoint (e.g., https://&lt;resource-name&gt;.cognitiveservices.azure.com). </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <param name="options"> The options for configuring the client. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public QuestionAnsweringClient(Uri endpoint, TokenCredential credential, QuestionAnsweringClientOptions options)
+        {
+            Argument.AssertNotNull(endpoint, nameof(endpoint));
+            Argument.AssertNotNull(credential, nameof(credential));
+            options ??= new QuestionAnsweringClientOptions();
+
+            Diagnostics = new ClientDiagnostics(options, true);
+            Pipeline = HttpPipelineBuilder.Build(options, new HttpPipelinePolicy[] { new BearerTokenAuthenticationPolicy(credential, "https://cognitiveservices.azure.com/.default") }, Array.Empty<HttpPipelinePolicy>(), new ResponseClassifier());
```

Comment:
Is this scope the same for all clouds, though?

Reply:
Good point, other clouds would have a different suffix for the endpoint.  Is there any example that you could point to for having different scope for different clouds?

Reply:
Great question, and I know [I'm guilty of it as well](https://github.com/Azure/azure-sdk-for-net/blob/4ee1f303c7c246de1b5da8f5467b1d6af2f75894/sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/ConversationAnalysisClient.cs#L66).

@schaabs @tg-msft do we have any patterns today that customize the scope for the `BearerTokenAuthenticationPolicy`?

Reply:
If the scope can be inferred from the endpoint you can automatically set the correct scope in the client constructor. If not you'll need to add an `Audience` property to your `ClientOptions` and provide an extensible enum with well known cloud audiences. 

You can use TextAnalytics as a guild. They added the [Audience](https://github.com/Azure/azure-sdk-for-net/blob/7091cb1413ae9639a2f620fbaceacbbd2ba014e2/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClientOptions.cs#L59) property to `TextAnalyticsClientOptions` and added the extensible enum [TextAnalyticsAudience](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsAudience.cs).

Reply:
Thanks for the suggestions, updated the PR.